pallets/crowdloan-claim/Makefile

```diff
@@ -0,0 +1,102 @@
+###############################################################################
```

Comment:
Is this something that can be extracted to the parent level? And reuse for the path provided as input, instead of having a copy of it within every pallet?

Reply:
I did not add that. @tankofzion what do you think, is this possible?

Reply:
Yes. It was more a helper for me. You can drop it

Reply:
Okay. Will remove it then.